### PR TITLE
Create 0280-wiggle-sort.java

### PR DESCRIPTION
**Pull request by dmgodoy**
[//]: # "Pull Request Template"
[//]: # "Replace the placeholder values in the template below"

- **File(s) Modified**: _0280-wiggle-sort.java_
- **Language(s) Used**: _java_
- **Submission URL**: _https://leetcode.com/problems/wiggle-sort/submissions/1443245145/_

[//]: # "Getting the Submission URL"
[//]: # "Go to the leetcode [`Submissions tab`](https://user-images.githubusercontent.com/71089234/180188604-b1ecaf90-bf27-4fd6-a559-5567aebf8930.png)"
[//]: # "and [click on the `Accepted` status of your submission.](https://user-images.githubusercontent.com/71089234/180189321-1a48c33f-aa65-4b29-8aaa-685f4f5f8c9e.png)]"
[//]: # "Finally copy the URL from the nav bar, it should look like https://leetcode.com/problems/[problem-name]/submissions/xxxxxxxxx/"

### Important
Please make sure the file name is lowercase and a duplicate file does not already exist before merging.

### CODE REVIEW

File: java/0280-wiggle-sort.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Solution {
+    public void wiggleSort(int[] nums) {
+        for(int i = 1; i < nums.length; i++){
+            if((i % 2 == 0 && nums[i] > nums[i - 1]) 
+            || (i % 2 == 1 && nums[i] < nums[i - 1]))
```
Comment (Ykhan799) [inline java/0280-wiggle-sort.java:5]: It's better to have a space after control flow statements. There should also be an additional space between the for loop and the opening bracket

File: java/0280-wiggle-sort.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+                swap(nums, i, i - 1);
+        }
+    }
+    private void swap(int[] nums, int i, int j){
+        int temp = nums[i];
```
Comment (Ykhan799) [inline java/0280-wiggle-sort.java:10]: Add a space before the opening bracket

**Comment by Ykhan799** [comment]
@dmgodoy Solution looks good. Please address my comments regarding spacing. The spaces will help improve readability of your code

File: java/0280-wiggle-sort.java (hunk 1 above)
Comment (dmgodoy) [reply java/0280-wiggle-sort.java]: Added spacing before the control flow statements

File: java/0280-wiggle-sort.java (hunk 2 above)
Comment (dmgodoy) [reply java/0280-wiggle-sort.java]: Added space before opening bracket

**Comment by dmgodoy** [comment]
Thanks for the feedback. I have just updated the code

**Pull request merged**